base/ca/shared/conf/CS.cfg

```diff
@@ -230,6 +230,7 @@ ca.scep.allowedHashAlgorithms=SHA256,SHA512
 ca.scep.encryptionAlgorithm=DES3
 ca.scep.allowedEncryptionAlgorithms=DES3
 ca.scep.nonceSizeLimit=16
+ca.scep.allowedDynamicProfileIds=caRouterCert
```

Comment:
This is not necessary since the code already defines the default value.

If we add this line, I'd suggest adding an upgrade script as well so new instances and upgraded instances are consistent.